src/network/mod.rs

```diff
@@ -283,16 +309,18 @@ fn connect_and_propagate_with<F>(
                 Ok(()) => (),
                 Err(e) => {
                     info!(
+                        state.logger(),
```

Comment:
This more properly belongs to the `conn_state` logger so that we'd have the address context?